src/cow-react/common/pure/RateInfo/index.tsx

```diff
@@ -136,7 +136,7 @@ export function RateInfo({
       )}
       <div>
         <RateWrapper onClick={() => setCurrentIsInversed((state) => !state)}>
-          <span title={currentActiveRate.toSignificant(18) + ' ' + rateInputCurrency.symbol}>
```

Comment:
that looks better. I rememeber seeing these and even commenting in some PR. Pls, let's make a search in case we are hardcoding toSignificant(18) or toFixed(18) somewhere else. 

In general we should avoid to do this "temporal" things, because then they are hard to find in tests. Well, Elena finds them, but that's not the point :)

Reply:
As far as I can see, `toSignificant(18)` is only used in one place to display active rate but it might be ok in that case.